packages/network/test/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
What's up with the huge amount of spacing?

Reply:
I'm using vscode's default JSON formatter, for consistency